docs/2-guides/2-island-architecture/README.md

```diff
@@ -1,4 +1,11 @@
-# Island Architecture
+---
+page:
+  $path: /sites/academy/home/documentation/jahia/8_2/developer/javascript-module-development/client-side-javascript
+  jcr:title: Client-side JavaScript
```

Comment:
Is it indended to update the title (and the page name) from _Island Architecture_ to _Client-side JavaScript_?
If that's the case, maybe we should rename the folder and update a bit the first sentence of this page ("this architectural pattern" is not really introduced)?